src/redux/api/slice.ts

```diff
@@ -173,7 +173,15 @@ export const apiSlice = createApi({
         body: patch,
       }),
     }),
-  }),
+    recurrenceRulesGenerateOrders: builder.mutation ({
+      query: date => ({
```

Comment:
I fixed a lot of warnings about passing non-serialisable data to redux when migrating to v2. I believe there should be a warning here as well. Is there? If there is we should pass a date as a string instead of a `moment`

Reply:
Done at https://github.com/coopcycle/coopcycle-app/pull/2027/commits/94a30c2237e435253bdb8336ed5efe76bbb98d03